extra-wm/i3status-rust/autobuild/build

```diff
@@ -0,0 +1,13 @@
+abinfo "Building binary"
```

Comment:
Please cut all portion before man page installation, and rename this file `beyond`.